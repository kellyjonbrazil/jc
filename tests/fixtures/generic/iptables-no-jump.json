[{"chain":"INPUT","rules":[{"target":null,"prot":"udp","opt":null,"source":"anywhere","destination":"anywhere"}]}]
