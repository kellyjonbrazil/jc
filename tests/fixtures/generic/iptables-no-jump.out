Chain INPUT (policy ACCEPT)
target     prot opt source               destination
           udp  --  anywhere             anywhere

